skyflow/service_account/_token.py

```diff
@@ -22,8 +22,11 @@ def generate_bearer_token(credentialsFilePath: str) -> ResponseToken:
     `credentialsFilePath` is the file path in string of the credentials file that is downloaded after Service Account creation.
 
     Response Token is a named tupe with two attributes:
-        1. AccessToken: The access token
-        2. TokenType: The type of access token (eg: Bearer)
+    1. AccessToken: The access token
+    2. TokenType: The type of access token (eg: Bearer)
```

Comment:
The response token is a named tuple with two attributes:
- `AccessToken`: The access token.
- `TokenType`: The type of the token. For example, "Bearer".